plugins/arDominionB5Plugin/scss/_fullwidthtreeview.scss

```diff
@@ -0,0 +1,78 @@
+@use "sass:math";
+
+// From css/fullWidthTreeView.css
```

Comment:
If you're copying the styles as they are, you could import directly the existing CSS file.

Reply:
They were modified a little - some are not necessary.